### PR TITLE
correct output format for classification models

### PR DESCRIPTION
**Pull request by vladimir-dudnik**
check and correct description for classification model's output format (probability in [0, 1] range and logits format)

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
cc @jkamelin @Anna-Mironova @Wovchena

**Review by eaidova — APPROVED** [review]
